js/repl/public/index.html

```diff
@@ -0,0 +1,52 @@
+<!doctype html>
```

Comment:
> In theory, the JS can be rebuilt entirely separately from the Jekyll build, if you can configure it to build it directly into the `_site` directory.

That's what the PR currently does (via `yarn build`) although nothing hooks into the `make build` process to run it.

> Yeah, Jekyll is slow...

I've been waiting for `npm start` to run for about 5 minutes and I'm stuck at "_Generating..._" 🤔